src/app/rpmostree-dbus-helpers.c

```diff
@@ -908,15 +908,15 @@ rpmostree_print_signatures (GVariant *variant,
           if (i != 0)
             g_string_append (sigs_buffer, sep);
 
-          gboolean isKeyMissing;
-          g_variant_get_child (v, OSTREE_GPG_SIGNATURE_ATTR_KEY_MISSING, "b", &isKeyMissing);
+          gboolean is_key_missing;
+          g_variant_get_child (v, OSTREE_GPG_SIGNATURE_ATTR_KEY_MISSING, "b", &is_key_missing);
 
           const char *fingerprint;
           g_variant_get_child (v, OSTREE_GPG_SIGNATURE_ATTR_FINGERPRINT, "&s", &fingerprint);
 
-          if(isKeyMissing)
+          if(is_key_missing)
             {
-              g_string_append_printf (sigs_buffer, "Key %s not available. Can't validate signature!\n", "XXX");
+              g_string_append_printf (sigs_buffer, "Can't check signature: public key %s not found\n", fingerprint);
```

Comment:
I tried to align the output with the verbose output of the OSTree which is: `Can't check signature: public key not found\n`. But I also wanted to include the fingerprint, since there might be more than one key given.